content/docs/introduction/comparison.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Comparison to alternatives
+sort_rank: 30
+---
+
+# Comparisons
+
+We are often asked how Xmidt+Webpa compares to some alternative.  It's a great
```

Comment:
It might be nice to explain what XMiDT and WebPA are briefly, to make it clear why here at the beginning we are saying "XMiDT+WebPA" but towards the end we're only talking about XMiDT.